### PR TITLE
Add notifications to alert AWS Team on failures and Release blocking Sig

### PR DESCRIPTION
**Pull request by hakuna-matatah**
- Add notifications to alert AWS Scalability Team on failures and Release blocking Sig
- Also add them onto release blocking testgrid dashboards

- Discussed in Sig-scalability meeting today and AWS team and Google team will collaborate on overlapping failures

### CODE REVIEW

**Comment by hakuna-matatah** [comment]
@dims @wojtek-t  PTAL

**Comment by dims** [comment]
/hold need one of the release managers to approve as well

/approve
/lgtm

cc @kubernetes/sig-release-leads @kubernetes/release-managers

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-ec2.yaml
Comment (xmudrii) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-ec2.yaml:65]: I see that this test has been failing since November 21. At this time, I don't think we should add that test to the master-informing board. It's going to confuse Release Team members and Release Managers. Similar, we shouldn't send alerts to `release-team@kubernetes.io` at this time.

**Review by xmudrii — CHANGES_REQUESTED** [review]

**Comment by hakuna-matatah** [comment]
@dims based on previous comment, PTAL again.

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**
